oneanddone/users/views.py

```diff
@@ -82,7 +82,10 @@ def get_context_data(self, **kwargs):
         all_attempts_finished = self.object.user.taskattempt_set.filter(state=TaskAttempt.FINISHED)
         paginator = Paginator(all_attempts_finished, 20)
         page = self.request.GET.get('page', 1)
-
+        user_profile = self.object
+        if user_profile.bugzilla_email:
+            activity_url = 'https://bugzilla.mozilla.org/page.cgi?'
+            'id=user_activity.html&action=run&from=-14d&who=%s' % (user_profile.bugzilla_email)
```

Comment:
This code does not work. `activity_url` ends up being set to just `https://bugzilla.mozilla.org/page.cgi?`. Did you test it to see if the link created was correct?